test/common/index.js

```diff
@@ -36,6 +36,7 @@ const testRoot = process.env.NODE_TEST_DIR ?
 
 const noop = () => {};
 
+exports.fixtures = fixturesDir;
```

Comment:
The idea is not about adding the `fixturesDir` to this file. It is about loading the fixtures in the specific test file and to use the explicit fixture function provided there. In this case `path`.

Reply:
Got it. Will push out the change with the recommendation provided.